test/python/project.py

```diff
@@ -0,0 +1,240 @@
+import os
+import re
+
+from utils import create_project
+from utils import recursive_listdir
+from utils import tarantool_enterprise_is_used
+
+
+CARTRIDGE_PACK_SPECIAL_FILES = {
+    # pre and post build hooks
+    'cartridge.pre-build',
+    'cartridge.post-build',
+
+    # deprecated files
+    '.cartridge.ignore',
+    '.cartridge.pre',
+}
+
+
+# ###############
+# Helpers
+# ###############
+def get_base_project_rocks(project_name, rockspec_name):
+    return {
+        '.rocks',
+        '.rocks/share/tarantool/rocks/manifest',
+        os.path.join('.rocks/share/tarantool/rocks', project_name),
+        os.path.join('.rocks/share/tarantool/rocks', project_name, 'scm-1'),
+        os.path.join('.rocks/share/tarantool/rocks', project_name, 'scm-1/rock_manifest'),
+        os.path.join('.rocks/share/tarantool/rocks', project_name, 'scm-1', rockspec_name),
+    }
+
+
+def filter_out_files_removed_on_pack(project_files):
+    # remove .git files and special files (pre and post build hooks)
+    def is_removed_on_pack(filename):
+        if filename in CARTRIDGE_PACK_SPECIAL_FILES:
+            return True
+
+        if filename.startswith('.git') and filename != '.gitignore':
+            return True
+
+    return set(filter(
+        lambda x: not is_removed_on_pack(x),
+        project_files
+    ))
+
+
+###############
+# Class Project
+###############
+class Project:
+    def __init__(self, name, basepath, template='cartridge'):
+        self.name = name
+        self.basepath = basepath
+        # create project and save its path
+        self.path = create_project(basepath, name, template)
+
+        # project files list
+        self.project_files = recursive_listdir(self.path)
+
+        # files that should be delivered in the result package
+        self.distribution_files = filter_out_files_removed_on_pack(self.project_files)
+        self.distribution_files.add('VERSION')
+        if tarantool_enterprise_is_used():
```

Comment:
You use this function twice in this ... function

May be let's store it to var and reuse